CONTRIBUTING.rst

```diff
@@ -77,7 +77,7 @@ copyright and the reference to the MIT License.
 
 5. Follow the Linux kernel coding style, which can be found in the file
    ``Documentation/process/coding-style.rst`` in the Linux kernel repository.
-   Note that the coding style isn't writen on stone, if there is a good reason
+   Note that the coding style isn't written on stone, if there is a good reason
```

Comment:
Just to be pedantic, I think this phrase normally goes ‘written *in* stone’ or ‘*set in* stone’, so it’d be nice to see that fixed at the same time 😊

Reply:
Okay. I will fix. :)